postgres/storage.go

```diff
@@ -9,8 +9,8 @@ type Storage struct {
 }
 
 // NewStorage creates a new storage
-func NewStorage(db *sqlx.DB) *Storage {
-	return &Storage{
+func NewStorage(db *sqlx.DB) Storage {
```

Comment:
ℹ️ Changed that just to follow the go convention of avoid returning pointers from local (function) variables